esteid_certificates/files/EID_Q_2024R.pem

```diff
@@ -1,77 +1,40 @@
-Subject: CN=TEST of SK ID Solutions EID-Q 2024R,organizationIdentifier=NTREE-10747013,O=SK ID Solutions AS,C=EE
```

Comment:
question: Out of curiosity, why did an existing cert change?

Reply:
Good question, it does seem that it changed from test certificate to live certificate with same name.

It does look like there was an error on SK side that was fixed.